cmd/bridge/main.go

```diff
@@ -197,15 +193,14 @@ func main() {
 		os.Exit(1)
 	}
 
-	capnslog.SetGlobalLogLevel(capnslog.INFO)
 	if *fLogLevel != "" {
-		llc, err := rl.ParseLogLevelConfig(*fLogLevel)
+		msg, err := glogSetter(convertLogLevel(*fLogLevel))
```

Comment:
I don't think we need a `glogSetter()` func in here since we aren't using `glog`.

That exists for systems that migrate from `glog` to `klog`.

Reply:
I think the name is just bad.  I still need to set the log level though.  I have tweaked it.